### ISSUE TITLE
didn't get the special battle chest reward from weekly quest

### ISSUE THREAD

**Issue by farhanajn** [ValveSoftware/Dota2-Gameplay#5692]
### Description

Not given a special battle chest for 13 in weekly quest 2

### Screenshots

![2022-11-27 (2)](https://user-images.githubusercontent.com/63141277/204126365-dcdca204-561e-4111-bdb1-d3736a1f75dc.png)
![image](https://user-images.githubusercontent.com/63141277/204126382-c9e2441e-917b-4f1f-b1c5-37f52e83e84e.png)
![2022-11-27 (3)](https://user-images.githubusercontent.com/63141277/204126386-15931482-45bb-4c62-a099-4c81a370ac68.png)

**Comment by TheUnholyWrath** [comment]
Have you checked your armory for a chest? Have you checked your Points Log to see if they were applied without the animation?

**Comment by farhanajn** [comment]
> Have you checked your armory for a chest? Have you checked your Points Log to see if they were applied without the animation?

yep, already checked armory there's none and already check point log only get 1 from weekly bonus which is 100 points
